pertpy/data/_datasets.py

```diff
@@ -1429,3 +1432,31 @@ def cinemaot_example() -> AnnData:  # pragma: no cover:
     adata = sc.read_h5ad(output_file_path)
 
     return adata
+
+
+def cinemaot_full() -> AnnData:  # pragma: no cover
+    """Complete CINEMA-OT dataset.
+
+    Ex vivo stimulation of human peripheral blood mononuclear cells (PBMC) with interferon. This is the full dataset
+    containing 9209 cells that were stimulated with IFNb, IFNg, IFNb+IFNg, or left unstimulated. A subsampled version
+    of the dataset is available via cinemaot_example().
+
+    References:
+        Preprint: https://doi.org/10.1101/2022.07.31.502173
+        Dataset available here: https://datadryad.org/stash/dataset/doi:10.5061/dryad.4xgxd25g1
+
+    Returns:
+        :class:`~anndata.AnnData` object of PBMCs stimulated with interferon.
+    """
+    output_file_name = "Integrated_subset.h5ad"
```

Comment:
Let's use a better output name here.

Can we use the authors name? We can ask Mingze if it's unclear who generated it...

Reply:
Based on what is written as a proposed citation [here](https://datadryad.org/stash/dataset/doi:10.5061/dryad.4xgxd25g1), I'd then propose _Dong_2023_?

Reply:
Yes!